src/module.ts

```diff
@@ -14,18 +15,29 @@ export default defineNuxtModule<ModuleOptions>({
   },
   defaults: {
     devOnly: false,
-    extensions: [".graphql", ".gql"]
+    extensions: [".graphql", ".gql"],
+    configPath: "codegen.yml",
   },
-  async setup({ devOnly, extensions }, nuxt) {
+  async setup({ devOnly, extensions, configPath }, nuxt) {
+    let config, cwd;
+
     // Run in development mode only
     if (devOnly && !nuxt.options.dev) {
       return;
     }
-    // Load GraphQL Code Generator configuration from rootDir
-    const { config, filepath } = await loadCodegenConfig({
-      configFilePath: nuxt.options.rootDir,
-    });
-    const cwd = dirname(filepath);
+
+    // Load GraphQL Code Generator configuration
+    try {
+      const configFilePath = join(nuxt.options.rootDir, configPath);
+      const { config: codegenConfig, filepath } = await loadCodegenConfig({
+        configFilePath,
+      });
+      config = { ...codegenConfig };
+      cwd = dirname(filepath);
+    } catch {
+      logger.error(`Unable to find ${configFilePath}`);
```

Comment:
I saw that some nuxt modules use a logger instance created with `.withScope('nuxt:[MODULE_NAME]')` but i'm not too familiar with how logging works in nuxt.